### PR TITLE
Add 404 error page to fellowship

### LINKED ISSUE #759: Page not found 404 notice for fellowship
**Is your feature request related to a problem? Please describe.**
In fellowship public UI when the id doesn't match an existing certificate, it renders a blank page.

**Describe the solution you'd like**
Need to render Page not found 404 notice when there is no certificate for the requested id.

**Additional context**
For additional details on the project please refer [this](https://sef.discourse.group/t/developing-an-admin-platform-for-sef-fellowship-program/20?u=gravewalker) thread.

### PR DESCRIPTION
**Pull request by nisan-abeywickrama**
## Purpose

The purpose of this PR is to fix #759

## Goals
Solving the issue of getting a blank page when the id doesn't match an existing certificate

## Approach
Adding a 404 error template and rendering it when the id doesn't match.

  
### Preview Link

https://pr-783-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/main/webapp/fellowship/index.html
Comment (Gravewalker666) [inline src/main/webapp/fellowship/index.html]: It's better if we could wrap this `h3` in some parent divs and add classes `container`, `row`, `col text-center` classes.

**Review by Gravewalker666 — CHANGES_REQUESTED** [review]
Awesome work @nisan-abeywickrama thank you so much for contributing. Please resolve this requested minor issues to get the approval.

File: src/main/webapp/fellowship/index.html
Comment (nisan-abeywickrama) [reply src/main/webapp/fellowship/index.html]: I added the above changes

**Review by Gravewalker666 — APPROVED** [review]
Brilliant work dude!!! The changes will get merged once another member approves it.

**Review by jayasanka-sack — APPROVED** [review]
Thanks for getting this done, @nisan-abeywickrama! 😍

You’ll be added to our GitHub leaderbord! 
https://sefglobal.org/github-leaderboard/

**Pull request merged**
